src/ensembl/production/metadata/grpc/client_examples.py

```diff
@@ -317,6 +318,11 @@ def get_genome_uuid_by_tag(stub):
     print("**** Genome Tag: foo ****")
     print(genome_uuid4)
 
+def get_ftp_links(stub):
+    request1 = FTPLinksRequest(genome_uuid="b997075a-292d-4e15-bfe5-23dca5a57b26", dataset_type="all")
+    links = stub.GetFTPLinks(request1)
+    print("**** FTP Links ****")
+    print(links)
 
```

Comment:
Can we have example here please? where where genome uuid and/or dataset_type are not valid
```suggestion
    request2 = FTPLinksRequest(genome_uuid="some-random-invalid-genome-uuid", dataset_type="all")
    links2 = stub.GetFTPLinks(request2)
    print("**** FTP Links (Invalid Genome UUID) ****")
    print(links2)
    request3 = FTPLinksRequest(genome_uuid="some-random-invalid-genome-uuid", dataset_type="foo")
    links3 = stub.GetFTPLinks(request3)
    print("**** FTP Links (Invalid Genome UUID and Dataset Type) ****")
    print(links3)
    request4 = FTPLinksRequest(genome_uuid="b997075a-292d-4e15-bfe5-23dca5a57b26", dataset_type="foo")
    links4 = stub.GetFTPLinks(request4)
    print("**** FTP Links (Valid Genome UUID and Invalid Dataset Type) ****")
    print(links4)
```

Reply:
Added now. thank you